src/engines/basic-render-engine.ts

```diff
@@ -24,25 +24,49 @@ const getPixelRatio = (ctx) => {
     return dpr / bsr;
 };
 
-export const defaultRenderSettings = {
-    timeUnits: 'ms',
-    styles: {
-        main: {
-            blockHeight: 16,
-            blockPaddingLeftRight: 4,
-            backgroundColor: 'white',
-            font: `10px sans-serif`,
-            fontColor: 'black',
-            tooltipHeaderFontColor: 'black',
-            tooltipBodyFontColor: '#688f45',
-            tooltipBackgroundColor: 'white',
-            headerHeight: 14,
-            headerColor: 'rgba(112, 112, 112, 0.25)',
-            headerStrokeColor: 'rgba(112, 112, 112, 0.5)',
-            headerTitleLeftPadding: 16,
-        },
-    },
+export type RenderOptions = {
+    tooltip?: Function | boolean;
```

Comment:
I am not really a fan of `Function` type
is there maybe a more narrowed type for tooltip?

Reply:
Yes, my mistake. I forgot about the tooltip function notation, we already have it. I'll fix it soon.
I think this is enough for the first time. Now we can move along the roadmap and try to implement something new.)

Reply:
I think we should actually be in a state where we can turn on the `"strict": true` in the tsconfig
that way we can be more safe when developing new stuff
I think there are places where also `"strictNullChecks": true` in tsconfig could cause some compile errors

but that is something for another PR